### PR TITLE
Feat: Buttons, Text, List Component

### PR DESCRIPTION
**Pull request by purohitdheeraj**
This Branch Contains :
- Button Component
- Text Utility
- List component

### CODE REVIEW

File: docs/main.css [hunk 1]
```diff
@@ -7,12 +10,21 @@
...
 }
 
 body {
   font-size: 1.6rem;
 }
 
+/* top to bottom */
+
+#float-btn{
+  /* display: none; */
```
Comment (iRohitGaur) [inline docs/main.css]: remove commented code

**Review by iRohitGaur — APPROVED** [review]
Good to merge with changes

File: library/components/text/text.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
+.h1,.h2,.h3,.h4,.h5,.h6{
+    margin: 0;
```
Comment (amartya-rs) [inline library/components/text/text.css:2]: Just a suggestion, you can include margin: 0; in reset css

**Comment by amartya-rs** [comment]
Rest looks good to me.

**Review by zoef-yt — APPROVED** [review]
With above mentioned changes.  Good to merge.

File: docs/main.css (hunk 1 above)
Comment (purohitdheeraj) [reply docs/main.css]: Thanks for pointing it out Rohit, I actually need that line for the bottom to Top floating button
was definitely going to miss that 😅

File: library/components/text/text.css (hunk 2 above)
Comment (purohitdheeraj) [reply library/components/text/text.css:2]: Hey Amartya, I did it intentionally for the heading component to remove the block-level and inline-level margins. 
My library will not have default reset styles (reset is just for the documentation page) , I want users to define it as per their preference.

Thanks for your suggestion 😃

File: library/style-guide.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* flex and column utility are used in alignment in library */
+@import url("../utils/utility.css");
+
+:root{
+    --RED :  #FF715B;
```
Comment (deekshithmd) [inline library/style-guide.css:5]: try to give color names based on the functionality rather than actual color name

**Pull request merged**